.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
That's intense !..

Reply:
In my mind, it could be every millisecond that I would not care either.
Dependabot open PR when needed (i.e. security derps).
If we have dependabots PR  too often is that we have crappy dependencies and culling is required.

TL;DR: Key thing is that Dependabot updates for security purposes, which is quite rare, so the interval is irrelevant.
A dev in my mind must review code daily, so having it daily makes sense. I only want to have security alerts and PR open  ASAP, just to be 'aware' of them at the very least. Daily is the most frequent, so here's that.

Reply:
Found you J, ain't needed anymore https://coveord.atlassian.net/wiki/spaces/CM/pages/1786708612/Package+Rollout+Certifier